Simple HTML parser

phorward opened the pull request:
Hello, this is a first version of a simple HTML parser that can be used to transform an HTML input source into a nested htmltree elements structure.
Maybe you find it useful. I didn't test it yet with all runtime environments htmltree should run within.

Michael-F-Ellis commented:
Thanks, Jan.  A parser could be a nice addition to htmltree.  I have a tight deadline to finish a project for a client this week so I won't get a chance to look at your contribution and test it until next week. Hope that's ok.

Kind regards,
Mike

phorward commented:
> Hope that's ok.
Sure it is, don't feel stressed with it in any way. I think this implementation won't work with Transcrypt so far. Transcrypt has so many pitfalls and problems to deal with right now...

phorward commented:
Hi @Michael-F-Ellis, I've just updated my HTML parser to work both with CPython 3.7 and Transcrypt 3.7.5.

I've put the pragmas above the docsctring because they produced different results with Transcrypt. Please feel free to change the section below around the "__main__" to fulfill previous standards, I've just changed it this way for direct testing.
